samples/cellular/nrf_cloud_multi_service/sample.yaml

```diff
@@ -84,7 +84,7 @@ tests:
       - nrf9151dk/nrf9151/ns
       - nrf9160dk/nrf9160/ns
       - nrf9161dk/nrf9161/ns
-    extra_args: nrf_cloud_multi_service_SHIELD=nrf7002ek
+    extra_args: nrf_cloud_multi_service_SHIELD=nrf7002ek_nrf7000
```

Comment:
There are some references to this shield in `` that I think also need to be updated. They are all in the `Building with nRF7002 Wi-Fi scanning support` section.

Specifically, lines 797, 804. These should, if I am not mistaken, switch to using the `nrf7002ek` shield.

But beware, any mentions of the shield outside this section are for connectivity-enabled builds, so I don't think those need to change.